dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/AnalyticsTableUpdateParams.java

```diff
@@ -124,6 +134,11 @@ public Date getLastSuccessfulUpdate()
         return lastSuccessfulUpdate;
     }
 
+    public boolean hasSkipPrograms()
```

Comment:
Use `isSkipPrograms` here, standard for boolean primitive methods.

Reply:
done